src/ProfileEndpoints.jl

```diff
@@ -61,49 +70,60 @@ controlled by `n=`. If you assume an average stack depth of 100, and you were ai
 The default `n` is 1e8, which should be big enough for most profiles.
 """
 
+function handle_cpu_profile(qp, stage_path = nothing)
+    # Run the profile
+    n = convert(Int, parse(Float64, get(qp, "n", default_n())))
+    delay = parse(Float64, get(qp, "delay", default_delay()))
+    duration = parse(Float64, get(qp, "duration", default_duration()))
+    with_pprof = parse(Bool, get(qp, "pprof", default_pprof()))
+    return _do_cpu_profile(n, delay, duration, with_pprof, stage_path)
+end
```

Comment:
It looks like before your PR the order in this file was:
- for each type of profile:
    - sync and async endpoint functions first
    - then the shared helpers for those functions after

Now, you've mixed in some helpers (the handle_ functions), and the order is mixed: you have the handle helpers before the endpoint and the rest of the helpers after.

I think it's generally important for readability to maintain a consistent ordering through the file. I think different people have different preferences about how to order functions, but I feel strongly that the best approach when modifying existing code is to follow the conventions already in use. 

Can you adjust the ordering to match the existing file ordering? Thanks